client/cypress/integration/authentication/sign-in.spec.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-named-as-default,import/no-named-as-default-member
+import SignInPage from '../../model/SignInPage';
+import { validUser } from '../../fixtures/users';
+// eslint-disable-next-line import/no-named-as-default,import/no-named-as-default-member
+import HomePage from '../../model/HomePage';
+
+const SIGN_OUT_LINK_TEXT = 'SIGN OUT';
+
+describe('Sign in page', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line no-undef
```

Comment:
Here you have a kind of specific of Cypress syntax. I believe Eslint does not understand that `cy` is global variable of Cypress.

2 ways to fix this issue:
- setup cypress plugin in config (highly recommended)
- let Eslint know that `cy` is a global variable